dataclients/kubernetes/endpoints.go

```diff
@@ -22,18 +22,18 @@ type endpointList struct {
 	Items []*endpoint `json:"items"`
 }
 
-func formatEndpointString(ip, protocol string, port int) string {
-	return protocol + "://" + net.JoinHostPort(ip, strconv.Itoa(port))
+func formatEndpointString(ip, scheme string, port int) string {
```

Comment:
This is not required by this PR but I noticed the inconsistency and decided to fix it along.